sdk/storage/azure-storage-blob/azure/storage/blob/aio/blob_service_client_async.py

```diff
@@ -286,17 +286,13 @@ def list_containers(
         :param bool include_metadata:
             Specifies that container metadata be returned in the response.
             The default value is `False`.
-        :param str marker:
-            An opaque continuation token. This value can be retrieved from the
-            next_marker field of a previous generator object. If specified,
-            this generator will begin returning results from this point.
         :param int results_per_page:
             The maximum number of container names to retrieve per API
             call. If the request does not specify the server will return up to 5,000 items.
         :param int timeout:
             The timeout parameter is expressed in seconds.
         :returns: An iterable (auto-paging) of ContainerProperties.
-        :rtype: ~azure.core.blob.models.ContainerPropertiesPaged
+        :rtype: ~azure.core.async_paging.AsyncItemPaged[~azure.core.blob.aio.models.ContainerProperties]
```

Comment:
I believe models are not "aio", and that should be azure.core.blob.models.ContainerProperties